src/cursor/cur_stat.c

```diff
@@ -95,6 +95,7 @@ __curstat_get_value(WT_CURSOR *cursor, ...)
 
     WT_ERR(cst->stats_desc(cst, WT_STAT_KEY_OFFSET(cst), &desc));
     if (F_ISSET(cursor, WT_CURSTD_RAW)) {
+        WT_ERR(__curstat_print_value(session, cst->v, &cst->pv));
```

Comment:
Why is this needed?

Reply:
It's possible to open a raw cursor on statistics (python uses 'raw' for all cursors).  A raw cursor returns a blob of all the arguments packed together.  In WT, we don't know which values will be used, so we need to blob them all.  And at this point in the code, the string version of the value has not been generated.

Reply:
Sounds good, thank you!

Reply:
@sueloverso, yes, that's correct.  Should there be a comment?